darwin/importer/importer.py

```diff
@@ -26,7 +25,7 @@
 
 if TYPE_CHECKING:
     from darwin.client import Client
-    from darwin.dataset import RemoteDataset
+    from darwin.dataset.remote_dataset import RemoteDataset
```

Comment:
Confused by this change? Was there a reason?

Reply:
Mypy complaint - not direct import